hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/file/FileSourceBuilder.java

```diff
@@ -98,7 +98,7 @@
      */
     public FileSourceBuilder(@Nonnull String path) {
         this.path = requireNonNull(path, "path must not be null");
-        if (!(Paths.get(path).isAbsolute() || hasHadoopPrefix(path))) {
+        if (!hasHadoopPrefix(path) && !Paths.get(path).isAbsolute()) {
```

Comment:
I find the double negation less readable.

Reply:
You can't read it aloud with the parenthesis, but _no dispute against taste_ (proti gustu žiaden dišputát :) ), reverting